roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -91,7 +91,7 @@ def run(self, terms, variables=None, zones_enabled=True, short_version=None,
                 {'name': 'InterPodAffinityPriority', 'weight': 1}
             ])
 
-        if short_version in ['3.5', '3.6']:
+        if short_version in ['3.5', '3.6', '3.7']:
```

Comment:
@aveshagarwal https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py#L104-L115 and https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py#L104-L115